tests/conftest.py

```diff
@@ -90,20 +90,31 @@ def postgres_db(postgresql_proc):
 
 
 @pytest.fixture(scope="function")
-async def create_db(postgres_db, database_name):
+async def create_db(postgres_db, database_name_internal):
+    """
+        This one is linked to eventloop fixture, which is function scoped. This kind of makes this messy
```

Comment:
This comment does not seem to contain the information required to understand this. Same for the database_name_internal fixture.